### DISCUSSION TITLE
Minimal Platine mit Jumper für Lautstärke.

### DISCUSSION THREAD

**Discussion by beabel** [beabel/radiowecker#51] [Ideen / Ideas]
Ich habe die Platine jetzt mal mit einem Jumper versehen um die Max Verstärkung anpassen zu können.

# Hinweis
Es ist mittlerweile **bestellt, und getestet worden**!

[Platinen Projekt](https://easyeda.com/editor#project_id=b7c90a5cbe554e4dbcf25dbf78662e11)

![PCB_0](https://github.com/beabel/radiowecker/assets/12296286/6daa2de3-399a-4701-9de8-90602cf77e4c)

**Comment by digi-thomas2003** [comment]
Hallo @beabel,
Deine Platine ist eine klasse Idee!
Wie kann ich diese bestellen, bereits bestückt (Widerstände, Q1, Jumper)?
Habe das bei EasyEDA nicht gefunden...
Danke!

**Comment by beabel** [comment]
hi @digi-thomas2003 
oben unter Fabrizierung:
![image](https://github.com/user-attachments/assets/e8fb7308-9d03-44b5-b006-28403db21104)
 bitte noch schauen, ob auch eine verbindung zwischen diesen punkten besteht (gelber Hinweis) bei mir mußte ich da noch eine Brücke auf der Rückseite löten.:
![image](https://github.com/user-attachments/assets/c133aa2d-8c74-4084-ac2f-93e5337b9b12)
